crates/pera-core/src/dwallet_mpc/mpc_manager.rs

```diff
@@ -199,8 +202,11 @@ impl DWalletMPCManager {
         Ok(())
     }
 
-    // todo(zeev): doc this.
-    pub fn get_decryption_share(&self) -> DwalletMPCResult<DecryptionKeyShare> {
+    /// Get the decryption share for the current party.
+    // This will be changed in #382
```

Comment:
I thought this pr intention is to "Use network DKG real cryptographic functions"?....

Reply:
It is, but only the class groups keygen functions. The DKG will be implemented after.